draft-ietf-httpbis-binary-message.md

```diff
@@ -0,0 +1,641 @@
+---
+title: "Binary Representation of HTTP Messages"
+abbrev: Binary HTTP Messages
+docname: draft-ietf-httpbis-binary-message-latest
+category: std
+ipr: trust200902
+area: ART
+workgroup: HTTP
+venue:
+  group: HTTP
+  type: Working Group
+  home: https://httpwg.org/
+  mail: ietf-http-wg@w3.org
+  arch: https://lists.w3.org/Archives/Public/ietf-http-wg/
+  repo: https://github.com/httpwg/http-extensions/labels/binary-messages
+
```

Comment:
Needs `github-issue-label` too. (we might want to put that under  `venue` eventually, but that can be done separately.